inference-engine/src/transformations/src/transformations/convert_opset1_to_legacy/convert_mul_add_to_scaleshift_or_power.cpp

```diff
@@ -138,24 +138,40 @@ void ngraph::pass::ConvertMulAddToScaleShiftOrPower::convert_mul_add_to_scaleshi
         const auto output_shape_rank = output_shape.rank().get_length();
 
         if (res1 == CONVERSION_RESULT::NONE || res2 == CONVERSION_RESULT::NONE ||
-            ((res1 == CONVERSION_RESULT::SCALE_SHIFT || res2 == CONVERSION_RESULT::SCALE_SHIFT) && output_shape_rank < 4)) {
+            ((res1 == CONVERSION_RESULT::SCALE_SHIFT || res2 == CONVERSION_RESULT::SCALE_SHIFT) &&
+            (output_shape_rank == 1 || output_shape_rank > 4))) {
             return false;
         }
 
+        bool is_dequantization =
+            (add_node->get_rt_info().count("DEQUANTIZATION") != 0 || mul_node->get_rt_info().count("DEQUANTIZATION") != 0);
+
         // TODO: in case if scale and shift constants has equal values the best way is to convert them to Power
-        if (res1 == CONVERSION_RESULT::SCALE_SHIFT || res2 == CONVERSION_RESULT::SCALE_SHIFT) {
+        if (res1 == CONVERSION_RESULT::SCALE_SHIFT || res2 == CONVERSION_RESULT::SCALE_SHIFT || is_dequantization) {
             NodeVector new_ops;
 
             auto weights_in = ngraph::op::util::normalize_constant(const_weights_node, output_shape);
             auto biases_in = ngraph::op::util::normalize_constant(const_bias_node, output_shape);
             new_ops.push_back(weights_in);
             new_ops.push_back(biases_in);
 
-            if (res1 == CONVERSION_RESULT::POWER) {
+            if (is_dequantization) {
+                const Shape data_shape = data_node.get_shape();
```

Comment:
If you need only rank and one dimension then do not call get_shape(). First check that rank is static then check number of dims and then take your dimension.